### ISSUE TITLE
Problems about low-accuracy in object detection

### ISSUE THREAD

**Issue by Whisper-Cao** [tensorflow/models#2916]
I train and evaluate pasval_vol_2012 dataset, with the pre-trained model generated by faster_rcnn_resnet101_coco_2017. I train the model for 99 steps but the evaluation result is rather poor. What's the problem? Should I try to choose another pre-trained model or train for more steps?
Morever, it seems only person can be detected. Why is this?

**Comment by aselle** [comment]
I apologize but I am having a hard time understanding what the problem is, where the problem is, and what version it affects. Please resubmit and pay attention to the issue template (https://github.com/tensorflow/tensorflow/issues/new) . Please provide all the information it asks. Thank you.
